Arcport buildsys patch

alexeysmirnovspb opened the pull request.

JaccovG committed:
Merge pull request #1 from tensorflow/master

downstream latest changes from tensorflow

JaccovG committed:
Add support for synopsys ARC based platforms

- add arc specific makefile include
- link mli lib
- Update makefile generator for TARGET_ARCH=arc
  In case TARGET_ARCH=arc: changing the makefile generator to add the CC/LD executables to the Makefile; diabling Mbed,Keil and Arduino projects generation, adding TCF_FILE variable support that also defines the TARGET string
- TCF file is copied to the project root directory to get rid of the absolute path
- adding .elf extension (ARC only); default TCF is ARC EM7D; clean target for project makefile template
- Linker output is redirected by the appropriate option
- Additn a suffix (TCF name) to the MLI directory name to support parallel project generation inside the same sources tree; using gmake to build MLI

JaccovG committed:
Merge pull request #2 from tensorflow/master

downstream latest changes from tensorflow

JaccovG committed:
Merge pull request #3 from tensorflow/master

downstream latest changes from tensorflow

JaccovG committed:
Merge pull request #4 from tensorflow/master

downstream latest changes from tensorflow

JaccovG committed:
Merge pull request #5 from foss-for-synopsys-dwc-arc-processors/arcport_final

First port to synopsys ARC processors

JaccovG committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	tensorflow/lite/micro/tools/make/helper_functions.inc
#	tensorflow/lite/micro/tools/make/third_party_downloads.inc

JaccovG committed:
Merge pull request #6 from tensorflow/master

downstream latest changes from tensorflow

JaccovG committed:
Fix merge issue

JaccovG committed:
Merge pull request #8 from tensorflow/master

downstream latest changes from tensorflow

JaccovG committed:
Merge branch 'master' into merge_first_PR

JaccovG committed:
Merge pull request #10 from foss-for-synopsys-dwc-arc-processors/merge_first_PR

Merge first pr

JaccovG committed:
Merge pull request #11 from foss-for-synopsys-dwc-arc-processors/newupstream

downstream latest changes from tensorflow

JaccovG committed:
Merge pull request #12 from foss-for-synopsys-dwc-arc-processors/callmli

Add calls to mli lib functions for ARC platform

JaccovG committed:
Use datamove in conv wrapper

JaccovG committed:
Merge pull request #13 from foss-for-synopsys-dwc-arc-processors/newupstreamJan14

downstream latest changes from tensorflow

JaccovG committed:
Merge remote-tracking branch 'origin/master' into move

JaccovG committed:
add data move functionality to depthwise, fc, pooling

Commit:
Minor fixes to restore 'generate_projects' target functionality

dzakhar committed:
person_detection example: wrap data with named bss section

JaccovG committed:
add LCF file for ARC target

JaccovG committed:
Fix warnings in ARC specific build

JaccovG committed:
Update URL to latest MLI lib with optimizations for person detect example

Commit:
Initial implementation of TCF and LCF files support for IoTDK and EMSDP platforms

JaccovG committed:
fix memory allocation issue for person detect example

JaccovG committed:
Merge pull request #15 from foss-for-synopsys-dwc-arc-processors/move

use MLI datamove in TF wrappers

JaccovG committed:
update MLI lib to performance optimized MLI1.1 pre-release

JaccovG committed:
update MLI lib to performance optimized MLI1.1 pre-release

JaccovG committed:
Merge branch 'master' of https://github.com/tensorflow/tensorflow into merge_upstream

JaccovG committed:
Fix for upstream merge conflict

the location of the header file was changed in the upstream archive.
but the makefile was not updated.

JaccovG committed:
Merge pull request #17 from foss-for-synopsys-dwc-arc-processors/merge_upstream

Merge upstream

gbaned commented:
@alexeysmirnovspb Thank you for your contribution. Can you please sign CLA and resolve the conflicts? Thanks!

JaccovG committed:
Add slicing logic for convolution layers

in case the tensors don't fit completely in local memory, slicing is used to split the tensors.

JaccovG committed:
Merge branch 'master' into slicing

JaccovG committed:
Merge pull request #18 from foss-for-synopsys-dwc-arc-processors/slicing

Slicing part 1

dzakhar committed:
Merge branch 'master' into arcport_buildsys_patch

dzakhar committed:
Updated LCF for EMSDP and fixes for arc build process

gbaned commented:
@alexeysmirnovspb it still shows CLA is pending , can you please sign CLA. Thanks!

alexeysmirnovspb commented:
Sorry, this pull request has been done by mistake. Closing it.
